solidity/test/BancorNetwork.js

```diff
@@ -18,14 +17,20 @@ const ContractFeatures = artifacts.require('ContractFeatures');
 const EtherToken = artifacts.require('EtherToken');
 const TestNonStandardERC20Token = artifacts.require('TestNonStandardERC20Token');
 
+const contractFeaturesId = web3.fromAscii('ContractFeatures');
```

Comment:
we can add these to a new helper util or just the utils we have

Reply:
Done